docs/releases/1.11.27.txt

```diff
@@ -0,0 +1,15 @@
+============================
+Django 1.11.27 release notes
+============================
+
+*Expected January 2, 2020*
+
+Django 1.11.27 fixes a data loss bug in 1.11.26.
+
+Bugfixes
+========
+
+* Fixed a data loss possibility in the
+  ``contrib.postgres.forms.SplitArrayField``. When using with
```

Comment:
* I'd drop the `the`. 
* `SplitArrayField` is doc'd so `:class:...`

Reply:
Thanks :+1: Updated.